app/controllers/posts_controller.rb

```diff
@@ -7,4 +7,32 @@ def index
   def show
     @post = Post.find(params[:id])
   end
+
+  def new
+    @post = Post.new
+  end
+
+  def create
+    @post = Post.new(post_params)
+    respond_to do |format|
+      format.html do
+        if @post.save
+          flash[:success] = 'Post saved successfully'
+          redirect_to user_post_path(current_user, @post)
+        else
+          flash.now[:error] = 'Error: Post could not be saved'
+          redirect_to new_user_post_path(current_user)
+        end
+      end
+    end
+  end
```

Comment:
So I created 3 posts for a user with different titles. I noticed that the titles I used to create each post are not the same as the ones reflected on the page. Every post is created with default titles like Post 1, Post 2, Posts 3, etc although those are not the titles I entered when creating each post.

![image](https://user-images.githubusercontent.com/38088818/192106602-d4630e26-51bb-4a1c-b264-256c1a47a6d1.png)

- Kindly look into and make sure that users can add custom titles for each post. You can either run `rails c` and use it to see if data is created properly or use the `db/seeds.rb` to enter data and then run `rails db:seed`.